dockers/docker-orchagent/supervisord.conf.j2

```diff
@@ -302,3 +311,20 @@ dependent_startup_wait_for=swssconfig:exited
 environment=ASAN_OPTIONS="log_path=/var/log/asan/fdbsyncd-asan.log{{ asan_extra_options }}"
 {% endif %}
 {%- endif %}
+
+{% if is_fabric_asic == 0 %}
+{% if dash_offload_enabled == 1 %}
+[program:dashoffloadmanager]
```

Comment:
Could you add a check to start dashoffloadmanager only when the binary exist? We can avoid depenencies of sonic-swss and buildimage PRs.

Reply:
I'm not sure it's a common approach. If I add it, I'd have to remove it in the future. As for a dependency, we can wait for all the other PRs to merge and submodules update before merging it, or I can even make it a draft until then. What do you think?